packages/atomizer/src/rules.js

```diff
@@ -3911,6 +3939,27 @@ module.exports = [
     },
     /**
     ==================================================================
+    WILL-CHANGE
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Will change',
+        matcher: 'Wc',
+        allowParamToValue: false,
+        styles: {
+            'will-change': '$0',
+        },
+        arguments: [
```

Comment:
will change has many args, do we need `allowParamToValue` ?

https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
I guess custom would work here?

Reply:
Changed it to `true` to make it easier to use. Custom var would still be needed for double values